### PR TITLE
Update README instructions.

### PR DESCRIPTION
**Pull request by scottdangelo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,92 +24,98 @@ Capture and analyze social media for a specified Twitter handle or hashtag and l
...
 
-1. If you do not already have a Bluemix account, [sign up here](https://console.ng.bluemix.net/registration).
+1. If you do not already have a Bluemix account, [sign up for Bluemix](https://console.ng.bluemix.net/registration).
 2. Download and install the [Cloud Foundry CLI](https://console.ng.bluemix.net/docs/cli/index.html#cli) tool.
-3. Log into Bluemix with our account.
-4. From the Application Dashboard, Create a new Application.
-  - On the left, select Apps > Cloudfoundry Apps.
-  - On the right, select SDK for Node.js.
+3. Log into Bluemix with your account.
+4. From the `Application Dashboard`, Create a new `Application`.
```
Comment (dolph) [inline README.md]: Create -> create

File: README.md [hunk 2]
```diff
@@ -19,92 +24,98 @@ Capture and analyze social media for a specified Twitter handle or hashtag and l
...
+  - On the left, select `Apps > Cloudfoundry Apps`.
+  - On the right, select `SDK for Node.js`.
   - Provide a unique name for your application.
-5. Once the application is created, go into the application and select Connections.
-6. Create the required services and bind them to the newly created application.
-7. Leave the Connections page open, as you will reference the credentials in the subsequent setup step.
+5. Once the application is created, go into the application and select `Connections`.
+6. Create the required services and bind them to the newly created application: `Watson Conversation`, `Natural Lanuguage Understanding`, `Tone Analyzer`, and `Cloudant NoSQL DB`.
+7. Leave the `Connections` page open, as you will reference the credentials in the subsequent setup step.
```
Comment (dolph) [inline README.md]: "subsequent setup step" -> "next step" ?

File: README.md [hunk 3]
```diff
@@ -19,92 +24,98 @@ Capture and analyze social media for a specified Twitter handle or hashtag and l
...
+
+#### Setup local Deployment
 
-## Load/Create the Conversation API workspaces
+> Explanation: You will create the  Bluemix services and configure them to use on a locally running server app.
 
-> Note: Although we do provide you with a sample classification workspace and dialog workspace.  The idea is that you create your own that is relevant to your domain.
+If you do not already have a Bluemix account, [sign up for Bluemix](https://console.ng.bluemix.net/registration).
```
Comment (dolph) [inline README.md]: region-specific links here and L105-L108

File: README.md [hunk 4]
```diff
@@ -138,33 +137,52 @@ npm run setup
...
 
-At this point you should have decided what Twitter handle you will be "listening" to.  The Twitter handle would be something other people would tweet to, for example `@united`, `@AmericanAir`, `@JetBlue` (Please note that these are CASE SENSITIVE) etc.  This value is what Twitter would use as trigger to send you the tweets.
-
-When you selected the `Twitter` option on the main menu of the setup utility, you will be required to enter the Twitter Tokens first.  Then the screen name you are listening to, the Conversation API workspace id for the classifications and finally the Chatbot screen name and Conversation API workspace for the Dialog implementation.
+At this point you should have decided what Twitter handle you will be "listening" to.  The intents for this journey are configured to work with a Twitter account for an airline, but you could create relevant intents for whichever business domain you wish. The Twitter handle would be something other people would tweet to, for example `@aircanada`.  This value is what Twitter would use as trigger to send you the tweets.
```
Comment (dolph) [inline README.md]: Perhaps clarify the use of "intents" here (that it's the service, not "the intention of this journey")

Comment (dolph) [reply README.md]: Maybe link [@aircanada](https://twitter.com/aircanada) ?

File: README.md [hunk 5]
```diff
@@ -138,33 +137,52 @@ npm run setup
...
 
-The chatbot screen name is your own Twitter account.  DO NOT USE THE SAME TWITTER SCREEN NAME YOU USED TO LISTEN TO.  This way, when you want to chat to the Chatbot via Twitter, you can use your own screen name for example `@mysuperbot blah blah blah` to send messages to your bot.
+NOTE>
```
Comment (dolph) [inline README.md]: Did you mean to use:

> Note: ?

`NOTE>` just renders in github markdown as text.

File: README.md [hunk 6]
```diff
@@ -138,33 +137,52 @@ npm run setup
...
+NOTE> 
+Due to the potential for a large volume of API calls, this Accelerator will work best with a paid subscription to Bluemix.  If you try and use this accelerator on an screen name that produces a lot of tweets, then you take the risk of using up your free allocation of API calls very quickly.  The Accelerator will suspend for 15 minutes listening to tweets if there are errors returned from the enrichment pipeline.  When the receiver on the UI is paused, it usually means you have exceeded your limit for the day.
 
-If you don't want to implement a Chatbot, then leave this blank, and the Chatbot won't be started.
+When you selected the `Twitter` option on the main menu of the setup utility, you will be required to enter the Twitter Tokens first.  Then the screen name you are listening to, the Conversation API workspace id for the classifications and finally the Chatbot screen name and Conversation API workspace for the Dialog implementation.
```
Comment (dolph) [inline README.md]: "selected" is suddenly past tense.

The last sentence here is a fragment - I'm not sure how to fix it / what the intended meaning was.

File: README.md [hunk 7]
```diff
@@ -138,33 +137,52 @@ npm run setup
...
 
 You sometimes want to populate your database with previous Tweets.  This Accelerator provides you with the ability to go back 7 days to search for tweets that match your "listen to" screen name.  It will do the enrichment and place the tweets into the database.
 
-## Running the app on Bluemix
+### 5. Install Dependencies
+
+The application requires the following software to be installed locally.
+
+1. [Node (6.9+)](https://nodejs.org) Application runtime environment
+2. [NPM (3.10+)](https://www.npmjs.com/get-npm) Server side dependency management
```
Comment (dolph) [inline README.md]: (nit) "server side" -> "server-side"

File: README.md [hunk 8]
```diff
@@ -138,33 +137,52 @@ npm run setup
...
+
+Run the following command, from the application folder, to install both the client and server dependencies.
+
+```
+npm install
+```
+
+### 6. Run the application
+
+Either `Run the app on Bluemix` or `Run the app locally`
```
Comment (dolph) [inline README.md]: Missing a period.

File: README.md [hunk 9]
```diff
@@ -238,7 +247,9 @@ This file must be updated with your service credentials before the application c
 }
 ```
 
-8. Once all the credentials are in place, the application can be starter with `npm run develop`.
+##### Local application start
+
+Once all the credentials are in place, the application can be starter with `npm run develop`.
```
Comment (dolph) [inline README.md]: "starter" -> "started"

File: README.md [hunk 10]
```diff
@@ -1,13 +1,18 @@
-# Social CRM Accelerator
 WORK IN PROGRESS - Complete code coming soon!
 
-## Overview
+# Monitor Twitter Feeds to better understand customer sentiment using Watson Conversation, Tone Analyzer, and Natural Language Understanding
```
[the remark this answers is not in the document]
Comment (scottdangelo) [reply README.md]: I always over-capitalize....

File: README.md [hunk 11]
```diff
@@ -1,13 +1,18 @@
...
 
-Capture and analyze social media for a specified Twitter handle or hashtag and let Watson analyze the content.
+In this journey our server application subscribes to a twitter feed as configured by the user. Each tweet received will be analyzed for emotional tone and sentiment, and the intent of the tweet will be determined by the Watson Conversation service. All data is stored in a Cloudant database, with the opportunity to store historical data as well. The information is presented in a Web UI as a series of graphs and charts.
```
[the remark this answers is not in the document]
Comment (scottdangelo) [reply README.md]: done

File: README.md [hunk 12]
```diff
@@ -1,13 +1,18 @@
...
 
-![](./accelerator/images/Social_CRM_Starter_Application.png)
+When the reader has completed this journey, they will understand how to:
 
-# Flow
+* Run an application that monitors a Twitter feed.
+* Send the tweets to Watson Tone Analyzer, Conversation, and Natural Language Understanding for processing and analysis.
+* Store the information in a Cloudant database.
+* Present the information in a nodejs web UI.
```
[the remark this answers is not in the document]
Comment (scottdangelo) [reply README.md]: done

File: README.md [hunk 13]
```diff
@@ -19,92 +24,98 @@ Capture and analyze social media for a specified Twitter handle or hashtag and l
...
 The setup is done in 3 primary steps.  You will download the code, setup the application and then deploy the code to Bluemix.  If you would like to run the code locally, there will be one more step to configure the credentials locally.
 
-## Downloading the code
+1. [Clone the repo](#1-clone-the-repo)
+2. [Twitter Requirements](#2-twitter-requirements)
+3. [Create Watson services with IBM Bluemix](#3-create-watson-services-with-ibm-bluemix)
+4. [Import the Conversation workspace](#4-import-the-conversation-workspace)
+5. [Configure credentials](#5-configure-credentials)
+6. [Install Dependencies](#6-install-dependencies)
+6. [Run the application](#6-run-the-application)
```
[the remark this answers is not in the document]
Comment (scottdangelo) [reply README.md]: Yeah, renders OK in atom, but I didn't check the link.

File: README.md [hunk 14]
```diff
@@ -19,92 +24,98 @@ Capture and analyze social media for a specified Twitter handle or hashtag and l
...
+* You can create a normal Twitter account on [Twitter](https://twitter.com/signup) or use an existing account.  It is required to provide a unique email id that isn't already associated with an existing Twitter account as well as a phone number to verify the account.
+* Once you have the Twitter account created and verified, log in to [Twitter Dev](https://apps.twitter.com/) and create an application.  
+* Select the Keys and Access Tokens tab and generate a Consumer Key and Secret.
+Keep this page open as you will need to use these tokens into setup procedure in the application later on.
 
-Once you have the Twitter account created and verified, log in to [Twitter Dev](https://apps.twitter.com/) and create an application.  Select the Keys and Access Tokens tab and generate a Consumer Key and Secret.
+### 3. Create Watson services with IBM Bluemix
 
-Keep this page open as you will need to use these tokens into setup procedure in the application later on.
+Either Setup the Bluemix Deployment or Setup Local Deployment
```
[the remark this answers is not in the document]
Comment (scottdangelo) [reply README.md]: Just trying to point out the 2 options. Period seems good.

File: README.md [hunk 15]
```diff
@@ -19,92 +24,98 @@ Capture and analyze social media for a specified Twitter handle or hashtag and l
...
 
-## Setting up Bluemix
+#### Setting up Bluemix Deployment
 
 > Explanation: You will create a placeholder application in Bluemix that connects to all the required services first.
 
-1. If you do not already have a Bluemix account, [sign up here](https://console.ng.bluemix.net/registration).
+1. If you do not already have a Bluemix account, [sign up for Bluemix](https://console.ng.bluemix.net/registration).
```
[the remark this answers is not in the document]
Comment (scottdangelo) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (scottdangelo) [reply README.md]: oops

File: README.md (hunk 2 above)
Comment (scottdangelo) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (scottdangelo) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (scottdangelo) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (scottdangelo) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (scottdangelo) [reply README.md]: done

File: README.md [hunk 16]
```diff
@@ -138,33 +137,52 @@ npm run setup
...
 
-### Searching for Tweets
+##### Searching for Tweets
 
 > Note: If you want to leverage this feature, you have to complete the local setup.
 
 You sometimes want to populate your database with previous Tweets.  This Accelerator provides you with the ability to go back 7 days to search for tweets that match your "listen to" screen name.  It will do the enrichment and place the tweets into the database.
 
-## Running the app on Bluemix
+### 5. Install Dependencies
```
[the remark this answers is not in the document]
Comment (scottdangelo) [reply README.md]: done

File: README.md (hunk 7 above)
Comment (scottdangelo) [reply README.md]: done

File: README.md (hunk 8 above)
Comment (scottdangelo) [reply README.md]: done

**Pull request merged**